packages/jupyter-ai/src/components/chat-settings.tsx

```diff
@@ -327,6 +363,87 @@ export function ChatSettings(props: ChatSettingsProps): JSX.Element {
         )}
       </Select>
 
+      {/* Completer language model section */}
+      <h2 className="jp-ai-ChatSettings-header">
+        Completer model
+        {props.completionProvider ? (
+          <UseSignal signal={props.completionProvider.settingsChanged}>
+            {(): JSX.Element => (
+              <CompleterSettingsButton
+                provider={props.completionProvider}
+                openSettings={props.openInlineCompleterSettings}
+                selection={clmProvider}
+              />
+            )}
+          </UseSignal>
+        ) : (
+          <CompleterSettingsButton
+            provider={null}
+            openSettings={props.openInlineCompleterSettings}
+            selection={clmProvider}
+          />
+        )}
+      </h2>
+      <Select
+        value={clmProvider?.registry ? clmProvider.id + ':*' : clmGlobalId}
+        label="Language model"
```

Comment:
The label of this select dropdown should match the heading, e.g. "Inline completion model".

Reply:
I don't think this is the best presentation - the labels become duplicates of headings, but done in a9dc569af411c90cba5782c61ca0293d880a2cd3.

Reply:
I think labels also serve a secondary purpose for being more accessible, but I agree it's strange how we duplicate the heading text in the label as well. If you have any suggestions for alternative designs, feel free to open an issue.

However, for this PR, we should stick to the overall look used by the rest of the Settings UI. We can address the appearance for all settings here in a separate PR.